lib/data_stores/mysql_advanced_data_store.py

```diff
@@ -382,11 +400,32 @@ def _BuildInserts(self, values):
 
     return [aff4_q, subjects_q, attributes_q]
 
-  def _ExecuteTransaction(self, transaction):
+  def _ExecuteQuery(self, query, args=None):
```

Comment:
Is it going to cause you problems if I change this to ExecuteQuery (drop the underscore?) It's incorrect as is because you access it from outside the class.  I'm going to fix this and the super call when I merge it to the internal code base.